libnmstate/prettystate.py

```diff
@@ -99,6 +106,12 @@ def order_state(state):
     return state
 
 
+def represent_unicode(dumper, data):
+    node = yaml.ScalarNode(tag=u'tag:yaml.org,2002:str',
```

Comment:
Perhaps explain what this is in the doctext and consider dropping the `node` variable (return directly), as it tells nothing interesting.

Reply:
done